### PR TITLE
Update code for pg13 pgjdbc java11

### PR DESCRIPTION
**Pull request by Jan-M**
Get rid of Java 11+ deprecated warnings in reflections and so on

### CODE REVIEW

File: src/main/java/org/zalando/sprocwrapper/proxy/SProcCallHandler.java [hunk 1]
```diff
@@ -41,8 +42,8 @@ private static String getSqlNameForMethod(final String methodName) {
     private RowMapper getRowMapper(SProcCall scA) {
         if (scA.resultMapper() != Void.class) {
             try {
-                return (RowMapper<?>) scA.resultMapper().newInstance();
-            } catch (final InstantiationException | IllegalAccessException ex) {
+                return (RowMapper<?>) scA.resultMapper().getDeclaredConstructor().newInstance();
+            } catch (final InstantiationException | IllegalAccessException | SecurityException | InvocationTargetException | IllegalArgumentException | NoSuchMethodException ex) {
```
Comment (vadeg) [inline src/main/java/org/zalando/sprocwrapper/proxy/SProcCallHandler.java:46]: Could it be replaced with just `Exception` here and in other updated places?

File: src/main/java/org/zalando/sprocwrapper/sharding/ShardedDataAccessException.java [hunk 2]
```diff
@@ -13,6 +13,8 @@
  */
 public class ShardedDataAccessException extends DataAccessException {
 
+    private static final long serialVersionUID = 1L;
```
Comment (vadeg) [inline src/main/java/org/zalando/sprocwrapper/sharding/ShardedDataAccessException.java:16]: Why we need it this? Is it serialized?

Comment (Jan-M) [reply src/main/java/org/zalando/sprocwrapper/sharding/ShardedDataAccessException.java:16]: vscode warning, just wanted it all green.

File: src/main/java/org/zalando/sprocwrapper/proxy/SProcCallHandler.java (hunk 1 above)
Comment (Jan-M) [reply src/main/java/org/zalando/sprocwrapper/proxy/SProcCallHandler.java:46]: i think the work is done now, so lets stick with it.

**Comment by Jan-M** [comment]
:+1:

**Comment by vadeg** [comment]
👍

**Pull request merged**